sdk/core/azure-core/test/nlohmann-json-test/CMakeLists.txt

```diff
@@ -0,0 +1,134 @@
+# download test data
```

Comment:
license header comment missing

Reply:
Not ours,  can we add our header anyway?

Reply:
Good question, I am not sure, but it's probably fine to not add our header.

That said, in general, we should be able to modify the files as we see fit. One thing I recommend is that you add a comment at the top of the file (at least for the build related files like this cmakelist.txt and the cmake-modules file) mentioning where this is copied from, with a permalink to the file from the nlohmann repo.

The other files under `/src/` can remain as is, unchanged.